webclient/src/components/views/ProjectList.vue

```diff
@@ -14,31 +14,39 @@
         <Project v-for="project in projects" :project="project" :key="project.ID" v-on:update="Update" />
       </div>
     </div>
+    <div v-if="!loading">
```

Comment:
so does this show the spinner *OR* the form -- but not both?

I think it might be a nicer experience if we can show the spinner overlayed on top of the page.

Reply:
Correct, right now based on the logic it shows either the spinner or the content. I can change it so the spinner appears over the content instead though. This particular feedback is what I was looking for with this initial PR submission :)

Reply:
Awesome! Let's take a look at how that looks. 

Currently, I'm playing around with different ways to get feedback to the user. Could you try both of the following, just to see how they end up looking:

1. Spinner overlay, faded black background behind it -- similar to how the modal on iteration completion is rendered
2. Spinner is placed in a modal with the background faded black (like iteration modal without text)